pkg/aspect/clean/clean.go

```diff
@@ -165,3 +189,457 @@ func (c *Clean) Run(_ *cobra.Command, _ []string) error {
 
 	return nil
 }
+
+func (c *Clean) reclaimAll() error {
+	sizeCalcQueue := make(chan bazelDirInfo, 64)
+	confirmationQueue := make(chan bazelDirInfo, 64)
+	deleteQueue := make(chan bazelDirInfo, 128)
+	sizeQueue := make(chan float64, 64)
+	errorQueue := make(chan error, 64)
+
+	var sizeCalcWaitGroup sync.WaitGroup
+	var confirmationWaitGroup sync.WaitGroup
+	var deleteWaitGroup sync.WaitGroup
+	var sizeWaitGroup sync.WaitGroup
+	var errorWaitGroup sync.WaitGroup
+
+	errors := errorSet{nodes: make(map[errorNode]struct{})}
+
+	// Goroutine for processing errors from the other threads
+	go c.errorProcessor(errorQueue, &errorWaitGroup, &errors)
+	errorWaitGroup.Add(1)
+
+	// Goroutines for calculating sizes of directories
+	for i := 0; i < 5; i++ {
+		go c.sizeCalculator(sizeCalcQueue, confirmationQueue, &sizeCalcWaitGroup)
+		sizeCalcWaitGroup.Add(1)
+	}
+
+	// Goroutines for deleting directories.
+	for i := 0; i < 8; i++ {
+		go c.deleteProcessor(deleteQueue, &deleteWaitGroup, sizeQueue, errorQueue)
+	}
```

Comment:
this isn't symmetric with the loop right before - doesn't it also need to add to the deleteWaitGroup?

or do the deletes only start filling in when the user has been prompted below? then maybe a comment would be useful here about why it's the only place we start goroutines but then don't add to a WaitGroup

Reply:
`deleteWaitGroup` is used slightly differently than the other wait groups which is why its not added here. I have added a comment